test/spec/MainViewManager-test.js

```diff
@@ -536,6 +536,31 @@ define(function (require, exports, module) {
                     expect(MainViewManager.getLayoutScheme()).toEqual({rows: 1, columns: 1});
                 });
             });
+            it("should switch pane when alt-w is pressed", function () {
+                runs(function () {
+                    MainViewManager.setLayoutScheme(1, 2);
+                });
+                runs(function () {
+                    MainViewManager.switchPaneUnitTest1To2();
```

Comment:
See above for my comment on how to handle these; the tests are valid but they test if `switchPaneUnitTest1To2` work, instead of testing if the keypress itself work ^^